### PR TITLE
Trim whitespace from the parsed JWT

### PR DESCRIPTION
**Pull request by bruth**
Since the JWT is read from a file that may be produced programmatically
(in one form or another), it is possible for there to be one or more
additional whitespace surrounding the JWT in the decorated file. This
simply ensure the whitespace is trimmed as is performed when parsing the
seed value.

Signed-off-by: Byron Ruth <b@devel.io>

### CODE REVIEW

**Review by derekcollison — APPROVED** [review]
LGTM

**Comment by bruth** [comment]
A new release will need to be made to update the Go client and CLI. Should we wait until #38 is merged?

**Comment by derekcollison** [comment]
Yes.

**Comment by derekcollison** [comment]
Should be tomorrow.

**Pull request merged**
